war/src/main/less/modules/app-bar.less

```diff
@@ -0,0 +1,32 @@
+.jenkins-app-bar {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin-bottom: 30px;
+  padding-bottom: 30px;
+  border-bottom: 2px solid var(--panel-border-color);
+
+  .jenkins-app-bar__content {
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    width: 100%;
+  }
+
+  .jenkins-app-bar__controls {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    margin-left: 30px;
+  }
+
+  &--no-border {
+    border-bottom: none;
+    margin-bottom: 0;
+  }
+
+  h1, h2, h3 {
+    margin: 0;
+    font-size: 30px;
```

Comment:
We should style heading sizes using the `.h1`, `.h2` classes, etc. Also, font-size should be in rems in any case.